alphalens/performance.py

```diff
@@ -400,7 +400,7 @@ def quantile_turnover(quantile_factor, quantile):
     return quant_turnover
 
 
-def factor_rank_autocorrelation(factor, time_rule='W', by_group=False):
+def factor_rank_autocorrelation(factor, period=1, time_rule=None, by_group=False):
```

Comment:
I didn't remove it because I wasn't sure why it was introduced in first place, but if you are happy to remove it I will do that.